src/misclibs.F

```diff
@@ -4143,14 +4143,14 @@ subroutine calcener(nstat,rstat,ruh,rvh,zh,rmh,pi0,th0,rho,ua,va,wa,ppi,tha,   &
         foo3(k) = foo3t
         foo4(k) = foo4t
       enddo
-      !$acc loop vector reduction(+:ek,ei,ep,le)
+      !$acc end parallel
+      !$acc parallel loop vector default(present) reduction(+:ek,ei,ep,le)
       do k=1,nk
         ek=ek+foo1(k)
         ei=ei+foo2(k)
         ep=ep+foo3(k)
         le=le+foo4(k)
       enddo
```

Comment:
better to add the `!$acc end parallel` directive

Reply:
Done